src/main/obsRecorder.ts

```diff
@@ -174,44 +177,132 @@ const setOBSVideoResolution = (res: Size, paramString: string) => {
+};
 
 /*
 * setupScene
 */
-const setupScene = (monitorIndex: number, baseResolution: Size, outputResolution: Size): IScene => {
+const setupScene = (options: RecorderOptionsType): IScene => {
+  const outputResolution = parseResolutionsString(options.obsOutputResolution);
+  let baseResolution: Size;
+
   setOBSVideoResolution(outputResolution, 'Output');
 
-  // Correct the monitorIndex. In config we start a 1 so it's easy for users. 
-  const monitorIndexFromZero = monitorIndex - 1;
-  console.info("[OBS] monitorIndexFromZero:", monitorIndexFromZero);
-  const selectedDisplay = displayInfo(monitorIndexFromZero);
-  if (!selectedDisplay) {
-    throw Error(`[OBS] No such display with index: ${monitorIndexFromZero}.`)
-  }
+  let videoSource: IInput;
 
-  setOBSVideoResolution(selectedDisplay.physicalSize, 'Base');
+  switch (options.obsCaptureMode) {
+    case 'monitor_capture':
+      // Correct the monitorIndex. In config we start a 1 so it's easy for users.
+      const monitorIndexFromZero = options.monitorIndex - 1;
+      console.info("[OBS] monitorIndexFromZero:", monitorIndexFromZero);
+      const selectedDisplay = displayInfo(monitorIndexFromZero);
+      if (!selectedDisplay) {
+        throw Error(`[OBS] No such display with index: ${monitorIndexFromZero}.`)
+      }
 
-  const videoSource = osn.InputFactory.create('monitor_capture', 'desktop-video');
+      baseResolution = selectedDisplay.physicalSize;
 
-  // Update source settings:
-  let settings = videoSource.settings;
-  settings['monitor'] = monitorIndexFromZero;
-  videoSource.update(settings);
-  videoSource.save();
+      videoSource = createMonitorCaptureSource(monitorIndexFromZero);
+      break;
+
+    case 'game_capture':
+      baseResolution = outputResolution;
+      videoSource = createGameCaptureSource();
+      break;
 
-  // A scene is necessary here to properly scale captured screen size to output video size
-  const scene = osn.SceneFactory.create('test-scene');
+    default:
+      throw Error(`[OBS] Invalid capture mode: ${options.obsCaptureMode}`);
+  }
+
+  setOBSVideoResolution(baseResolution, 'Base');
+
+  const scene: IScene = osn.SceneFactory.create('test-scene');
   const sceneItem = scene.add(videoSource);
   sceneItem.scale = { x: 1.0, y: 1.0 };
 
+  console.log(`[OBS] Configured video input source with mode '${options.obsCaptureMode}'`, inspectObject(videoSource.settings))
+
+  watchVideoSourceSize(sceneItem, videoSource, baseResolution);
+
   return scene;
 }
 
+/**
+ * Create and return a game capture video source ('game_capture')
+ */
+const createGameCaptureSource = (): IInput => {
+  const videoSource = osn.InputFactory.create('game_capture', 'Game Capture');
+  const settings = videoSource.settings;
+
+  settings['capture_cursor'] = true;
+  settings['capture_mode'] = 'window';
+  settings['allow_transparency'] = true;
+  settings['priority'] = 1; // Window title must match
+  settings['window'] = 'World of Warcraft:GxWindowClass:Wow.exe';
```

Comment:
Are all wow versions the same here? Classic/PTR/Beta? How can we confirm?

Reply:
`priority` = 1 means it only considers "World of Warcraft" and as the window title, and doesn't look at "Wow.exe". I tested it for Classic now and it works fine.

If Beta/PTR hs a different Window title then it can't work unless that is exposed as a setting which would make it much less automatic.

![image](https://user-images.githubusercontent.com/875107/195981591-b16c8863-de9d-4e67-b4af-1c949d0d0c59.png)